.github/actions/passwords-tool/script.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+users=( admin kibanaserver kibanaro logstash readall snapshotrestore wazuh_admin wazuh_user )
+api_users=( wazuh wazuh-wui )
+
+echo '::group:: Change indexer password, password provided'
+
+bash wazuh-passwords-tool.sh -u admin -p LN*X1v.VNtCZ5sESEtLfijPAd39LXGAI
+ if ! curl -XGET https://localhost:9200/ -u admin:LN*X1v.VNtCZ5sESEtLfijPAd39LXGAI -k -w %{http_code} | grep "200"; then
```

Comment:
Why don't use = 401 instead of ! 200?

Reply:
Done https://github.com/wazuh/wazuh-packages/pull/1737/commits/054ac37040357937607329818f7f2548254c86cc